integration_tests/pages/shared/showReferral/risksAndNeeds/offenceAnalysis.ts

```diff
@@ -0,0 +1,83 @@
+import { CourseUtils, OffenceAnalysisUtils } from '../../../../../server/utils'
+import Page from '../../../page'
+import type { Course, OffenceDetail } from '@accredited-programmes/models'
+
+export default class OffenceAnalysisPage extends Page {
+  offenceDetail: OffenceDetail
+
+  constructor(args: { course: Course; offenceDetail: OffenceDetail }) {
+    const { course, offenceDetail } = args
+
+    const coursePresenter = CourseUtils.presentCourse(course)
+
+    super(`Referral to ${coursePresenter.nameAndAlternateName}`)
+
+    this.offenceDetail = offenceDetail
+  }
+
+  shouldContainBriefOffenceDetailsSummaryCard() {
+    cy.get('[data-testid="brief-offence-details-summary-card"]').then(summaryCardElement => {
+      this.shouldContainKeylessSummaryCard(
+        'Section 2.1 - Brief offence details',
+        OffenceAnalysisUtils.textValue(this.offenceDetail.offenceDetails),
+        summaryCardElement,
+      )
+    })
+  }
+
+  shouldContainImpactAndConsequencesSummaryList() {
+    cy.get('[data-testid="impact-and-consequences-summary-list"]').then(summaryListElement => {
+      this.shouldContainSummaryListRows(
+        OffenceAnalysisUtils.impactAndConsequencesSummaryListRows(this.offenceDetail),
+        summaryListElement,
+      )
+    })
+  }
```

Comment:
Do we want to test the card heading here too (and in other list methods on this page)? The test ID can be put on the card via the Nunjucks macro to facilitate this